src/Http/Http.Extensions/src/RequestDelegateFactory.cs

```diff
@@ -1445,8 +1551,9 @@ private class FactoryContext
         public bool DisableInferredFromBody { get; init; }
 
         // Temporary State
-        public ParameterInfo? JsonRequestBodyParameter { get; set; }
+        public ParameterInfo? RequestBodyParameter { get; set; }
         public bool AllowEmptyRequestBody { get; set; }
+        public bool IsRawRequestBody => RequestBodyParameter?.ParameterType == typeof(ReadOnlySequence<byte>);
```

Comment:
It'd be interesting to see if people actually start using it. It's a somewhat obtuse type to actually use (particularly when you can bind the PipeReader instead).

Reply:
It’s something we’d have to advertise. I had support for ReadOnlyMemory\<byte\> and byte[] but removed them.